modules/azure/keyvault.go

```diff
@@ -0,0 +1,182 @@
+package azure
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+
+	kvauth "github.com/Azure/azure-sdk-for-go/services/keyvault/auth"
+	kvmng "github.com/Azure/azure-sdk-for-go/services/keyvault/mgmt/2016-10-01/keyvault"
+	"github.com/Azure/azure-sdk-for-go/services/keyvault/v7.0/keyvault"
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/stretchr/testify/require"
+)
+
+// KeyVaultSecretExists indicates whether a key vault secret exists; otherwise false
+func KeyVaultSecretExists(t *testing.T, keyVaultName string, secretName string) bool {
+	result, err := KeyVaultSecretExistsE(keyVaultName, secretName)
+	require.NoError(t, err)
+	return result
+}
+
+// KeyVaultKeyExists indicates whether a key vault key exists; otherwise false.  This function would fail the test if there is an error.
+func KeyVaultKeyExists(t *testing.T, keyVaultName string, keyName string) bool {
+	result, err := KeyVaultKeyExistsE(keyVaultName, keyName)
+	require.NoError(t, err)
+	return result
+}
+
+// KeyVaultCertificateExists indicates whether a key vault certificate exists; otherwise false.
+// This function would fail the test if there is an error.
+func KeyVaultCertificateExists(t *testing.T, keyVaultName string, certificateName string) bool {
+	result, err := KeyVaultCertificateExistsE(keyVaultName, certificateName)
+	require.NoError(t, err)
+	return result
+}
+
+// KeyVaultCertificateExistsE indicates whether a certificate exists in key vault; otherwise false.
+// This function would fail the test if there is an error.
```

Comment:
This comment needs to be updated to remove the last line since this is the `E` version. This applies to all the `E` functions added here.